carrot-core/src/main/java/org/bigbase/carrot/redis/CarrotMain.java

```diff
@@ -42,36 +48,33 @@ private static void startNodes() {
       int port = Integer.parseInt(parts[1].trim());
       nodeServers[i] = new CarrotNodeServer(host, port);
       nodeServers[i].start();
+      log.debug("Start nodeServers '{}'}", nodeServers[i]);
```

Comment:
Remove last '}'

Reply:
Removed in 222_add_logging branch. This branch for maven compilation warning only.